bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+
+![Status Diagram](bip-update-process/status-diagram.png "Status Diagram for the BIP Workflow")
+
+### Ideation
+
+After having an idea, the author should evaluate whether it is meets the criteria to become a BIP, as described in
+this BIP. The idea must be of interest to the broader community or relevant to multiple software projects. Small
+improvements and matters concerning only a single project usually do not require standardization and should instead be
+brought up to the relevant project directly.
+
+The author should first research whether an idea has been considered before. Ideas in Bitcoin are often rediscovered,
+and prior related discussions may inform the author what issues may arise in its progression. After some investigation,
+the novelty of an idea can be tested by posting about it to the [Bitcoin development mailing
+list](https://groups.google.com/g/bitcoindev).
+
+Vetting an idea publicly before investing the time to formally describe the idea is meant to save both the author and
+the broader community time. Not only may someone point out relevant discussion topics that were missed in the author’s
+research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
+tests whether it is of interest to more people beside the author. After establishing that the idea may be of interest
+to the Bitcoin community, the author should work on drafting a BIP.
+
+It is recommended that complicated proposals be split into separate BIPs that each focus on a specific component of the
+overall proposal.
+
+### Progression through BIP Statuses
+
+As the first draft is taking shape, the author should present it to the [Bitcoin development mailing
+list](https://groups.google.com/g/bitcoindev). This gives the author a chance to collect initial feedback and address
+fundamental concerns. If the author wishes to work in public on the draft at this stage, it is recommended that they
+open a pull request against their personal fork of the BIPs repository instead of the main BIPs repository.
+
+The following sections refer to BIP Status Field values. The BIP Status Field is defined in the Header Preamble
+specification above.
+
+#### Preliminary
+
+After fleshing out the draft further and ensuring that it is of high quality and properly formatted, the author should
+open a pull request to the [BIPs repository](https://github.com/bitcoin/bips). The draft BIP must adhere to the
+formatting requirements specified below and named with a working title of the form "bip-title". The author must not
+self-assign a number to the BIP draft.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed
+beyond the ideation phase, e.g. by generating substantial public discussion and commentary from diverse contributors, by
+independent Bitcoin projects working on adopting the proposal, or by the author working for an extended period of time
```

Comment:
```suggestion
independent Bitcoin projects working on adopting the proposal, or by the author working for an extended time
```

Reply:
I don’t think this matches what I’m trying to express.